tasks/eth/1-security-council-phase-0/input.json

```diff
@@ -10,7 +10,7 @@
     {
       "to": "0x543bA4AADBAb8f9025686Bd03993043599c6fB04",
       "value": "0",
-      "data": "0xf2fde38b000000000000000000000000dc53b1d440b8a56dd50a6d69312dac903d3aa48b",
+      "data": "0xf2fde38b00000000000000000000000027d3C6Ac002fc5B5fAcf44cAa1C63d978b4A7e60",
```

Comment:
I ran:
```
$ cast 4byte-decode 0xf2fde38b00000000000000000000000027d3C6Ac002fc5B5fAcf44cAa1C63d978b4A7e60
1) "transferOwnership(address)"
0x27d3C6Ac002fc5B5fAcf44cAa1C63d978b4A7e60
```
to verify that the data here matches what's shown in the simulation. Will defer to you about whether this is worth adding as a validation step